packages/twenty-front/src/pages/settings/ai/components/SettingsAIAgentTableRow.tsx

```diff
@@ -0,0 +1,71 @@
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import { ReactNode } from 'react';
+
+import { TableCell } from '@/ui/layout/table/components/TableCell';
+import { TableRow } from '@/ui/layout/table/components/TableRow';
+import { Tag } from 'twenty-ui/components';
+import { useIcons } from 'twenty-ui/display';
+import { Agent } from '~/generated-metadata/graphql';
+
+export type SettingsAIAgentTableRowProps = {
+  action: ReactNode;
+  agent: Agent;
+  link?: string;
+};
+
+export const StyledAIAgentTableRow = styled(TableRow)`
+  grid-template-columns: 1fr 120px 36px;
+`;
+
+const StyledNameTableCell = styled(TableCell)`
+  color: ${({ theme }) => theme.font.color.primary};
+  gap: ${({ theme }) => theme.spacing(2)};
+  min-width: 0;
+  overflow: hidden;
+`;
+
+const StyledNameLabel = styled.div`
```

Comment:
looks like we should be re-using a UI component here (like TextWithToolTip or something like this)